backend/src/ee/controllers/v1/ssoController.ts

```diff
@@ -156,7 +157,10 @@ export const updateSSOConfig = async (req: Request, res: Response) => {
                     }
                 },
                 {
-                    authProvider: ssoConfig.authProvider
+                    authProviders: [ssoConfig.authProvider],
+                    $unset: {
+                        authProvider: 1
+                    }
```

Comment:
no need for this one

Reply:
I think instead of `$unset` on `authProvider` here, we set `authMethods` to `AuthMethod.EMAIL`. Thoughts?